src/priority_queue.erl

```diff
@@ -214,10 +216,28 @@ highest({queue, [], [], 0})     -> empty;
 highest({queue, _, _, _})       -> 0;
 highest({pqueue, [{P, _} | _]}) -> maybe_negate_priority(P).
 
+member(_X, {queue, [], [], 0}) ->
```

Comment:
What's the purpose of this function? When is something a "member" of a priority queue?